### PR TITLE
primer commit del seutUp

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary
subo el setup del proyecto

## Screenshot
captura de pantalla 2018-09-05 a la s 16 52 31

## Trello Card
https://trello.com/c/HPwzsXKd/11-setup-del-proyecto

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <h1 class="title">Hello World</h1> 
+</body>
+</html>
```
Comment (mmamaniwolox) [inline index.html]: Add a tailing line at the end of the file. Always in html and css

File: scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
+  }
```
Comment (mmamaniwolox) [inline scss/application.scss]: Add a tailing line

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: scss/application.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.title{
```
Comment (mmamaniwolox) [inline scss/application.scss]: Add an space between title and the `{`

File: scss/application.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+  color:red;
```
Comment (mmamaniwolox) [inline scss/application.scss]: Add an space between `color:` and `red`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Name both branch and commits in english. Everything must be in english :P

**Review by mmamaniwolox — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**